data/json/items/generic.json

```diff
@@ -1527,52 +1527,52 @@
     "flags" : ["MC_MOBILE", "MC_RANDOM_STUFF", "MC_MAY_BE_ENCRYPTED", "MC_TURN_USED"],
     "weight": 5,
     "volume": 0
-  },
-  {
-    "type":"GENERIC",
-    "id": "mobile_memory_card_used",
-    "symbol": ",",
-    "color": "white",
-    "name": "SD-Memory card (clean)",
-    "name_plural": "SD-Memory cards (clean)",
-    "description": "This memory card is either unused or has been wiped clean.  You could use it to store your data, though!",
-    "price": 500,
-    "price_postapoc": 0,
-    "material": "plastic",
-    "flags" : ["MC_MOBILE", "MC_USED"],
-    "weight": 5,
-    "volume": 0
-  },
-  {
-    "type":"GENERIC",
-    "id": "mobile_memory_card_encrypted",
-    "symbol": ",",
-    "color": "white",
-    "name": "SD-Memory card (encrypted)",
-    "name_plural": "SD-Memory cards (encrypted)",
-    "description": "This memory card appears to have the firmware encryption set.  Hopefully it contains something worth encrypting.",
-    "price": 2000,
-    "price_postapoc": 0,
-    "material": "plastic",
-    "flags" : ["MC_MOBILE", "MC_ENCRYPTED", "MC_TURN_USED"],
-    "weight": 5,
-    "volume": 0
-  },
-  {
-    "type":"GENERIC",
-    "id": "mobile_memory_card_science",
-    "symbol": ",",
-    "color": "white",
-    "name": "Science SD-Memory card",
-    "name_plural": "Science SD-Memory cards",
-    "description": "This memory card appears to be related to 'XEDRA', and is certainly encrypted.  Looks * Interesting *, though...",
-    "price": 10000,
-    "price_postapoc": 0,
-    "material": "plastic",
-    "flags" : ["MC_MOBILE", "MC_ENCRYPTED", "MC_TURN_USED", "MC_SCIENCE_STUFF"],
-    "weight": 5,
-    "volume": 0
-  },
+    },
+    {
```

Comment:
This formatting is not related to the main aim of this PR. It makes your PR bigger and excessively hard to review with no actual reason.